libs/form/src/next/input-field/stories/vl-input-field.stories-doc.mdx

```diff
@@ -29,6 +29,39 @@ import { VlInputFieldComponent } from '@domg-wc/form/next/input-field';
 
 <FormControlPublicMethods/>
 
+## Pattern validatie
+
+Pattern validatie kan op 2 manieren gebeuren:
+- Via het `pattern` attribuut
+- Via de `regex` property
+
+Het `pattern` attribuut kan gebruikt worden voor eenvoudige validatie.<br/>
+De `regex` property kan gebruikt worden voor complexere validatie.
+
```

Comment:
Eventueel nog toevoegen welke ValidityState key getriggerd word bij deze validatie gelijkaardig zoals je deed bij `vl-input-field-masked`.

Zouden we het verschaffen van die info ook mogelijks doortrekken voor de andere attributen die overeenkomen met specifieke ValidityState keys? die info kan ofwel eventueel bij de Storybook-`args` komen of ergens anders op een centrale plaats. Heb deze comment ook toegevoegd in toekomstige jira task ivm documentatie - kunnen dat dan nog altijd laten vallen.

Reply:
Is het goed als we dit op een ander punt opnemen?
We zouden ergens meer informatie moeten aanbieden ivm de standaarden die we volgen, het gebruik van de open-wc/form-participation package en de ElementInternals API.

Reply:
Is goed. Ik had ivm ValidityState al reeds toegevoegd in [UIG-2842](https://www.milieuinfo.be/jira/browse/UIG-2842). Heb jouw comment nu hier ook nog in toegevoegd. Dan nemen we dit later op.